server/gql/models/vehicles/index.js

```diff
@@ -121,3 +122,16 @@ export const vehicleMutations = {
   type: GraphQLVehicle,
   model: db.vehicleModel
 };
+
+function _formatForOrderByClause(args) {
+  const arr = args.split(',');
+  const orderByClause = [];
+  arr.forEach(element => {
+    const queryArgs = element.trim().split(' ');
+    const orderByCol = queryArgs[0];
+    const orderByType = queryArgs[1];
+    const query = 'queryColumn'.replace('queryColumn', orderByCol);
```

Comment:
What does this do ?

Reply:
I mean

```
 const query = 'queryColumn'.replace('queryColumn', orderByCol);
```

Reply:
The query is for generating literal query for Sequelize function, so I am using quotes on queryColum as the query requires the quotes to work. So I am replacing 'queryColumn' with something like `driver.user.distance_in_km`. 

This creates a query like : ORDER BY `driver.user.distance_in_km` ASC in SQL.